js/models/roomcollection.js

```diff
@@ -28,7 +28,9 @@
 
 	var RoomCollection = Backbone.Collection.extend({
 		model: OCA.SpreedMe.Models.Room,
-		comparator: 'name',
+		comparator: function(model) {
+			return -(model.get('lastPing'));
```

Comment:
The thing is that rooms get sorted by last room usage with the code above (maybe it's just magic 😄 ).